pkg/kubelet/cm/dra/state/cdi.go

```diff
@@ -23,7 +23,7 @@ limitations under the License.
 // Long term it would be good to avoid this duplication:
 // https://github.com/container-orchestrated-devices/container-device-interface/issues/97
 
-package dra
```

Comment:
This doesn't feel like it belongs in the state package since we will never be storing any state about the annotations. This should be associated with whatever type wraps the state in the main `dra` package and hangs additional functions / metadata off of it.